### PR TITLE
successfully added rainfall prediction using linear regression

### LINKED ISSUE #9: can i create a new pull request?
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Ashlin21dev**
my second decent pull request, i hope that i have done it better than the forst time!!

### CODE REVIEW

**Comment by Ashlin21dev** [comment]
can you sir please review it and accept it!!

File: Rainfall-prediction/datacleaning.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# importing libraries
+import pandas as pd
+import numpy as np
```
Comment (aravindvnair99) [inline Rainfall-prediction/datacleaning.py:3]: You can remove this line as it's not used anywhere.

File: Rainfall-prediction/raindata.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
+# importing libraries
+import pandas as pd
+import numpy as np
+import sklearn as sk
```
Comment (aravindvnair99) [inline Rainfall-prediction/raindata.py:4]: You can remove this line as it's not used anywhere.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@Ashlin21dev Remove the unused import statements.

File: Rainfall-prediction/datacleaning.py (hunk 1 above)
Comment (Ashlin21dev) [reply Rainfall-prediction/datacleaning.py:3]: i will do that

File: Rainfall-prediction/raindata.py (hunk 2 above)
Comment (Ashlin21dev) [reply Rainfall-prediction/raindata.py:4]: on it!

File: Rainfall-prediction/datacleaning.py (hunk 1 above)
Comment (aravindvnair99) [reply Rainfall-prediction/datacleaning.py:3]: @Ashlin21dev only `import numpy as np`

File: Rainfall-prediction/raindata.py (hunk 2 above)
Comment (aravindvnair99) [reply Rainfall-prediction/raindata.py:4]: @Ashlin21dev Only `import sklearn as sk`

Comment (Ashlin21dev) [reply Rainfall-prediction/raindata.py:4]: i ran into a problem sir!!

**Comment by Ashlin21dev** [comment]
i mistakenly deleted the previously yorked Repo

File: Rainfall-prediction/raindata.py (hunk 2 above)
Comment (aravindvnair99) [reply Rainfall-prediction/raindata.py:4]: @Ashlin21dev What's the problem?

**Comment by Ashlin21dev** [comment]
so i am not able to switch branches to edit!!, its a mistake from my side

**Comment by aravindvnair99** [comment]
> i mistakenly deleted the previously yorked Repo

@Ashlin21dev You can fork again and copy the changes from here to there. After that, send in your PR with proper title and description.

**Comment by Ashlin21dev** [comment]
like i have to totally create a new pull request isn't it?

**Comment by aravindvnair99** [comment]
> like i have to totally create a new pull request isn't it?
> 

@Ashlin21dev Yes, that's right. Because otherwise you won't be able to commit anything.

**Comment by Ashlin21dev** [comment]
sure sir then i will commit a new PR with those removed and with proper description

**Comment by Ashlin21dev** [comment]
sir i pulled a new PR can you please review it!!?

**Comment by aravindvnair99** [comment]
Closing for #14.

**Pull request closed without merging**